src/app.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Form from './form'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      question: '',
+      answer: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSave = this.handleSave.bind(this)
+  }
+  handleChange(event) {
```

Comment:
You can delete all methods that you were passing to your form, but don't need anymore.